modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc

```diff
@@ -40,20 +41,23 @@ Person:p -((LIKES>|<HAS_CREATOR):e)- Message:m
 
 === Edge type wildcards
 
-The underscore `\_` is a wildcard meaning any edge type.
-Arrowheads are used to indicate direction, e.g.,  `>` or `<` or `_`.
+Use the underscore `\_` or the keyword `ANY` as a wildcard meaning "any edge type."
```

Comment:
I'd qualify it earlier. Use the underscore `\_` or the keyword `ANY` to denote any undirected edge type.